company/views.py

```diff
@@ -413,9 +413,9 @@ def post(self,request,company,*args,**kwargs):
             return redirect("add_domain",company)
         
         parsed_url = urlparse(domain_data["url"])
-        domain = parsed_url.hostname
+        domain = (parsed_url.hostname).replace("www.","")
 
-        domain_data["url"] = f"{parsed_url.scheme}://{domain}" # clean the domain eg https://mail.google.com/mail1# -> https://mail.google.com  
+        # domain_data["url"] = f"{parsed_url.scheme}://{domain}" # clean the domain eg https://mail.google.com/mail1# -> https://mail.google.com  
```

Comment:
Remove the comment

Reply:
@just-ary27  Those scripts are using django templating to get the data, if i move them to seprate js file then it cannot fetch data using templating. I will move some of the script code to js file which are not using templating